spring-cloud-stream/pom.xml

```diff
@@ -35,6 +35,10 @@
 			<artifactId>spring-integration-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-jmx</artifactId>
```

Comment:
If you don't like this dependency, consider to use `@EnableIntegrationManagement`: http://docs.spring.io/spring-integration/reference/html/system-management-chapter.html#_configuring_metrics_capture